tutorials/Tutorial10_Knowledge_Graph.py

```diff
@@ -0,0 +1,95 @@
+import logging
+import subprocess
+import time
+from pathlib import Path
+
+from haystack.graph_retriever.text_to_sparql import Text2SparqlRetriever
+from haystack.knowledge_graph.graphdb import GraphDBKnowledgeGraph
+from haystack.preprocessor.utils import fetch_archive_from_http
+
+logger = logging.getLogger(__name__)
+
+
+def tutorial10_knowledge_graph():
+    # Let's first fetch some triples that we want to store in our knowledge graph
+    # Here: exemplary triples from the wizarding world
+    doc_dir = "../data/tutorial10_knowledge_graph/"
```

Comment:
lets rename and use further down when importing ttls etc

Reply:
done